integration/shared/isolated/remove_network_policy_command_test.go

```diff
@@ -133,15 +133,12 @@ var _ = Describe("remove-network-policy command", func() {
 					Eventually(session).Should(Say("OK"))
 					Eventually(session).Should(Exit(0))
 
-					session = helpers.CF("curl", fmt.Sprintf("/networking/v1/external/policies?id=%s", sourceAppGUID))
+					session = helpers.CF("network-policies")
+					Eventually(session).Should(Say(`Listing network policies in org %s / space %s as %s\.\.\.`, sourceOrg, sourceSpace, username))
+					Consistently(session).ShouldNot(Say("OK"))
```

Comment:
See comment in `add_network_policy_command_test.go.`

Reply:
Agreed, will remove.